i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md

```diff
@@ -1,143 +1,140 @@
-# 部署方式
+# Deployment method
 
-<!-- ## 集群搭建 -->
-
-## 单Master模式
+## Single Master mode
 
 :::caution
-这种方式风险较大，一旦Broker重启或者宕机时，会导致整个服务不可用。不建议线上环境使用, 可以用于本地测试。
+This mode carries a higher risk, as a restart or failure of the Broker will result in the entire service being unavailable. It is not recommended for use in online environments, but can be used for local testing.
 :::
 
-### 启动 NameServer
+### Start NameServer
 
 ```bash
-### 首先启动Name Server
+### Start Name Server first
 $ nohup sh mqnamesrv &
  
-### 验证Name Server 是否启动成功
+### Verify if the Name Server has started successfully
 $ tail -f ~/logs/rocketmqlogs/namesrv.log
 The Name Server boot success...
 ```
 
-### 启动 Broker
+### Start Broker
 
 ```bash
-### 启动Broker
+### Start Broker
 $ nohup sh bin/mqbroker -n localhost:9876 &
 
-### 验证Name Server 是否启动成功，例如Broker的IP为：192.168.1.2，且名称为broker-a
+### Verify that the Name Server has started successfully, for example, the broker IP is 192.168.1.2, and the name is broker-a
 $ tail -f ~/logs/rocketmqlogs/Broker.log 
 The broker[broker-a, 192.169.1.2:10911] boot success...
 ```
 
-## 多Master模式
-
-一个集群无Slave，全是Master，例如2个Master或者3个Master，这种模式的优缺点如下：
+## Multiple master mode
 
-- 优点：配置简单，单个Master宕机或重启维护对应用无影响，在磁盘配置为RAID10时，即使机器宕机不可恢复情况下，由于RAID10磁盘非常可靠，消息也不会丢（异步刷盘丢失少量消息，同步刷盘一条不丢），性能最高；
+there are no slaves in the cluster and all are masters, such as 2 masters or 3 masters. The advantages and disadvantages of this mode are as follows:
```

Comment:
there are no slaves in the cluster and all are masters, such as 2 masters or 3 masters.

->

The advantages and disadvantages of a cluster which is full of masters without slaves (e.g. 2 or 3 masters) is as follows: